released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.13" & < "8.16~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.13.0" & < "1.15~") | (= "dev") }
+  "coq-mathcomp-fingroup" { (>= "1.13.0" & < "1.15~") | (= "dev") }
+  "coq-mathcomp-algebra" { (>= "1.13.0" & < "1.15~") | (= "dev") }
+  "coq-mathcomp-solvable" { (>= "1.13.0" & < "1.15~") | (= "dev") }
+  "coq-mathcomp-field" { (>= "1.13.0" & < "1.15~") | (= "dev") }
+  "coq-mathcomp-finmap" { (>= "1.5.1" & < "1.6~") | (= "dev") }
+  "coq-mathcomp-bigenough" { (>= "1.0.0") }
+  "coq-hierarchy-builder" { (>= "1.2.0") }
```

Comment:
@affeldt-aist I thought one of the points of the release was to support HB 1.0 and 1.1? Can you confirm you have now switched to >= HB 1.2 completely?

Reply:
Let me check.

Reply:
I remember that we decided to switch to hb >= 1.2.0, I PRed this change but it was merged by Cyril who knows better, so that should be ok (ref: https://github.com/math-comp/analysis/pull/590).